lib/rails_admin_import/formats/csv_importer.rb

```diff
@@ -0,0 +1,56 @@
+require 'csv'
+
+module RailsAdminImport
+  module Formats
+    class CSVImporter
+      Formats.register(:csv, self)
+
+      # Default is to downcase headers and add underscores to convert into attribute names
+      HEADER_CONVERTER = lambda do |header|
+        header.parameterize.underscore
+      end
+
+      def initialize(import_model, params)
+        if params.has_key?(:file)
+          @filename = params[:file].tempfile
+        end
+        @import_model = import_model
+        @header_converter = import_model.config.header_converter || HEADER_CONVERTER
+      end
+
+      attr_reader :filename, :error
+
+      def valid?
+        if filename.nil?
+          @error = I18n.t('admin.import.missing_file')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.